test-app/src/test/kotlin/uk/gov/justice/digital/hmpps/testapp/integration/service/PrisonApiServiceTest.kt

```diff
@@ -6,10 +6,12 @@ import com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo
 import org.junit.jupiter.api.Test
 import uk.gov.justice.digital.hmpps.testapp.integration.IntegrationTestBase
 import uk.gov.justice.digital.hmpps.testapp.integration.wiremock.PrisonApiExtension.Companion.prisonApi
+import uk.gov.justice.hmpps.test.kotlin.auth.HmppsAuthApiExtension.Companion.hmppsAuth
 
 class PrisonApiServiceTest : IntegrationTestBase() {
   @Test
   fun `should supply authentication token`() {
+    hmppsAuth.stubGrantToken()
```

Comment:
these now need stubbing explicitly as they are using auth to get a token to call Prison API.  I think it is clearer this way as it is showing what happens when calls are made out using client credentials